samples/backend/python/quart/requirements.txt

```diff
@@ -0,0 +1,3 @@
+openai
+quart[dotenv]
+azure.identity
```

Comment:
Does azure.identity work? I always use azure-identity

Reply:
this works but I'll change it to azure-identity